gateway/assets/script/bootstrap.js

```diff
@@ -130,20 +129,21 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$timeout', '$md
                 url: "/system/functions",
                 data: $scope.item,
                 method: "POST",
-                headers: { "Content-Type": "application/json"},
-                responseType: "json"
+                headers: { "Content-Type": "application/json" },
+                responseType: "text"
```

Comment:
+                responseType: "text" ?

Reply:
`w.Write([]byte("Deployment error: " + err.Error()))` writes plain text and not json, if responseType is set to json then angular attempts to parse it and the data ends up being null

Reply:
OK